tools/FakeItEasy.Build/Program.cs

```diff
@@ -115,6 +116,10 @@
 
 RunTargetsAndExit(args, messageOnly: ex => ex is NonZeroExitCodeException);
 
+[SuppressMessage(
+    "StyleCop.CSharp.MaintainabilityRules",
+    "SA1400:AccessModifierMustBeDeclared",
+    Justification = "Cannot supply access modifier outside namespace.")]

```

Comment:
What do you mean? We can absolutely specify an access modifier.
Declaring it `public` causes `CA1050`, but `internal` or `file` works fine.
I think it would be better than suppressing the warning.

Reply:
I'm a moron. I tried `private`. Will amend.